### DISCUSSION TITLE
VLC Streaming outside home network (remote location)

### DISCUSSION THREAD

**Discussion by Slingboxforever** [GerryDazoo/Slinger#471] [Q&A]
Hi All,

I'm happily up and running with config / VLC on computer/laptop.   Can someone share steps to stream VLC on iOS phone outside my home network?  

Also, do I need to be running the server (streaming on home computer) to do this when I'm streaming outside home?

thanks!

**Comment by GerryDazoo** [comment]
Many people have issues with VLC on iOS. Check some of the threads here.
The short answer to your second question is "it depends". Please review the network architecture document that should have been downloaded with all the other code. The server can be running on any host that has Internet access. It does not have to be on your home LAN.

**Comment by DoogieHowitzerWMD** [comment]
VLC can work on iOS, but you need to tweak a few settings to get things working properly on a mostly consistent basis. A few people with iOS devices have found PlayerXtreme to work much more readily.

And as Gerry said, the server needs to actively be running in order to access the streams. Wherever the device running your server is in relation to the slingboxes, you need to set things up in the config file accordingly.
